librz/util/sdb/README.md

```diff
@@ -0,0 +1,79 @@
+SDB (string database)
```

Comment:
I think, this file could be removed as well.

Reply:
I'd keep it, for reference and because it provides some explanation about sdb